test-series/1/test-1.0.0.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<p>What is meant by "Only auto-baseline pairs were used for power spectrum estimation"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
It means that the power spectrum estimate was obtained by only cross-multiplying a visibility with itself, rather than averaging over all pairs of visibilities within a redundant group. (See Eq. 4 in [Parsons 2012](https://arxiv.org/pdf/1304.4991.pdf).)